snowflake/connect.go

```diff
@@ -133,10 +146,14 @@ func DSN(ctx context.Context, account, user,
 	}
 
 	config := gosnowflake.Config{
-		Account: account,
-		User:    user,
-		Region:  region,
-		Role:    role,
+		Account:     account,
+		User:        user,
+		Region:      region,
+		Role:        role,
+		Database:    database,
+		Schema:      schema,
+		Warehouse:   warehouse,
+		Application: "STEAMPIPE",
```

Comment:
I'm assuming this shouldn't be `STEAMPIPE` for all users? What is the `Application` used for? Do we need a config arg for it?